utils/cq/words.dict

```diff
@@ -238,6 +239,7 @@ libgurt
 libioil
 libs
 libvirt
+linux
```

Comment:
From memory Linux is accepted so it's probably better to change any code from linux to Linux and not make this change.

Reply:
Done. Thanks